exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,8 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var percentageOfStudents = Math.round(15*100/23) + '%';
+var percentageOfMentors = Math.round(8*100/23) + '%';
+
+console.log(percentageOfStudents);
+console.log(percentageOfMentors);
```

Comment:
Hi Johnny,
I have a little suggestion :)
You could add variables `numberOfStudents` and `numberOfMentors` and save the result to a new variable like `total` and then use all these variables in `Math.round()` instead of just using numbers(15, 8, 23).  Something like this: `Math.round(numberOfStudents*100/total)`.  
This will make your code less hard-coded, so you can change numberOfStudents or numberOfMentors to any other numbers - percentage would still be calculated correctly 😉

Also, I believe the expected result was "Percentage students: 65%
Percentage mentors: 35%".